src/ManageCourses.Api/Services/Data/DataService.cs

```diff
@@ -426,7 +421,8 @@ private Course LoadCourse(IReadOnlyList<UcasCourse> courseRecords)
                 Address4 = courseRecord.UcasCampus.Addr4,
                 PostCode = courseRecord.UcasCampus.Postcode,
                 ApplicationsAcceptedFrom = courseRecord.CrseOpenDate,
-                Code = courseRecord.UcasCampus.CampusCode
+                Code = courseRecord.UcasCampus.CampusCode,
+                Status = courseRecord.Status
```

Comment:
Add the trailing commas to lists to make diffs cleaner in future.

Minor.

Reply:
Done